app/ui/legacy/src/main/res/layout/message_compose_content.xml

```diff
@@ -120,14 +120,15 @@ the 'composer theme' setting).
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_marginEnd="8dp"
-                    android:background="@drawable/btn_edit" />
+                    app:icon="@drawable/ic_edit" />
```

Comment:
`ImageButton` requires the use of `android:src` or `app:srcCompat`. However, the intention was probably to switch to `MaterialButton`.

Suggestion:
```kotlin
<com.google.android.material.button.MaterialButton
    style="@style/Widget.Material3.Button.IconButton.Filled"
    android:id="@+id/quoted_text_edit"
    android:contentDescription="@string/message_compose_description_edit_quoted_text"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    app:icon="@drawable/ic_edit" />

<com.google.android.material.button.MaterialButton
    style="@style/Widget.Material3.Button.IconButton.Filled"
    android:id="@+id/quoted_text_delete"
    android:contentDescription="@string/message_compose_description_delete_quoted_text"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    app:icon="@drawable/ic_close" />
```

|Before|After|
|-|-|
|![image](https://github.com/thunderbird/thunderbird-android/assets/218061/d4707e25-92ce-4882-99b9-f1933e82c61a)|![image](https://github.com/thunderbird/thunderbird-android/assets/218061/45b938d7-8a34-44e1-8095-e5e848e61d9a)|